x-pack/plugins/observability_solution/apm/server/routes/services/get_service_instance_container_metadata.ts

```diff
@@ -54,6 +55,7 @@ export const getServiceInstanceContainerMetadata = async ({
   const response = await infraMetricsClient.search({
     size: 1,
     track_total_hits: false,
+    fields: ['*'],
```

Comment:
According to [ServiceInstanceContainerMetadataDetails](https://github.com/elastic/kibana/blob/b79855d0e96c444b06537a98c371c3c808c4ceac/x-pack/plugins/observability_solution/apm/server/routes/services/get_service_instance_container_metadata.ts#L26), this only requires the `kubernetes.*` fields